LIS3MDL.class.nut

```diff
@@ -179,7 +180,7 @@ class LIS3MDL {
             "y_negative" : interruptByte & 0x08 ? true : false,
             "z_negative" : interruptByte & 0x04 ? true : false,
             "overflow"   : interruptByte & 0x02 ? true : false,
-            "interrupt"  : interruptByte & 0x01 ? true : false
+            "interrupt"  : interruptByte & 0x01
```

Comment:
interrupt can be set as either high or low, so interrupt true is not always true when the state of the pin is 1.  I didn't have an elegant way to sort this out, so i'm just passing the state of the pin and the user can sort it out.  If you have a better solution i'm open to suggestions.